source/common/regular-expressions.js

```diff
@@ -57,25 +353,71 @@ module.exports = {
       'i' + flag
     )
   },
+
   /**
-   * Returns a regular expression that matches file IDs as in the settings
-   *
-   * @param   {boolean}  multiline  Whether to match multiline
+   * Returns a regular expression that matches MarkDown Tables.
    *
    * @return  {RegExp}              The compiled Regular Expression
    */
-  'getIDRE': function (multiline = false) {
-    let flag = (multiline) ? 'm' : ''
-    let idRegExpString = global.config.get('zkn.idRE')
-    // Make sure the ID definitely has at least one
-    // capturing group to not produce errors.
-    if (!(/\(.+?\)/.test(idRegExpString))) {
-      idRegExpString = `(${idRegExpString})`
-    }
+  'getTableRE': function () {
+    return RegExp(
+      /^\|.+\|$/.source,
+      'i')
+  },
 
+  /**
+   * Returns a regular expression that matches inline frames.
```

Comment:
Are you sure that's what this RegExp does? :D

Reply:
Yes. 100%. The function name is merely obfuscation.